src/components/footer/examples/footer-warning/index.njk

```diff
@@ -0,0 +1,14 @@
+---
+"fullWidth": true
+---
+{% from "components/footer/_macro.njk" import onsFooter %}
+{{
```

Comment:
Add lines: 

Use this if you need to show sensitive information on a post-submission page, such as the user's address or a response ID.  

The link should clear the any `sessionStorage` and send the user to a page outside of the transaction.